spec/java/util/zip/CRC32.lsl

```diff
@@ -0,0 +1,29 @@
+libsl "1.1.0";
+
+library std
+    version "11"
+    language "Java"
+    url "https://github.com/openjdk/jdk11/blob/master/src/java.base/share/classes/java/util/zip/CRC32.java";
+
+// imports
+
+import java/lang/Object;
+import java/util/zip/Checksum;
+
+
+// primary semantic types
+
+@implements("java.util.zip.Checksum")
+@public type CRC32
+    is java.util.zip.CRC32
+    for Object
```

Comment:
`CRC32` *can* be assigned to `Object` but, more interestingly, it can be assigned to `Checksum` which is more useful.
*(import for `Object` will become irrelevant then)*